### ISSUE TITLE
Doc page styling is not High Contrast friendly

### ISSUE THREAD

**Issue by MikhailArkhipov** [Microsoft/RTVS-docs#68]
VSTS 402678: Accessibility: MAS42A: High Contrast: Vertical separator/Grid Splitter when focused using keyboard is not identifiable in HC black

Ex, background does not change

![image](https://cloud.githubusercontent.com/assets/12820357/24723116/0e0f81d6-19fb-11e7-9c0b-47413a2eed61.png)

**Comment by smortaz** [comment]
It's going to be moved to docs.msft.com soon btw

**Comment by MikhailArkhipov** [comment]
VSTS 402680: Accessibility: MAS42A: High Contrast: 'R Tools for Visual Studio' and 'Home' links in LHS pane are not in compliance with HC themes
